web/pdf_page_view.js

```diff
@@ -128,7 +128,15 @@ var PDFPageView = (function PDFPageViewClosure() {
         div.removeChild(node);
       }
       div.removeAttribute('data-loaded');
+      div.setAttribute('data-pageremoved-triggered', true);
```

Comment:
@yurydelendik Do we need such attributes? It seems unnecessary to me as one can simply wait until the event fires...